CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: Contributing to UofTCoders
+---
+
+## Running a lesson
+
+### Making the lesson
+
+The Mozilla Science Study Group
+handbook [**here**](https://mozillascience.github.io/studyGroupHandbook/lessons.html#reuse)
+and [**here**](http://mozillascience.github.io/studyGroupHandbook/event-types.html#workalong)
+has several very good points about making a lesson. This section summarizes bits
+of the handbook, but also adds pieces that are missing from it.
+
+From the handbook:
+
+- **Keep it short and simple**
+- **Keep in mind the beginners**
+- **Don't use slides and focus on writing the code**
+
+Expanding on this are other things to keep in mind.
+
+**Data**: External data *should not be used* unless it is absolutely necessary. 
+Instead, rely on built-in or sample datasets provided by the respective 
+programming language or package.
+
+**Code**: This is the main thing that should be emphasized. How the code is
+used, it's specific applications, and it's meaning should be the entire focus of
+the lesson. Here are some tips:
+
+- Keep the code as *simple* as possible while still covering the concept. The
+session is, after all, only one hour.
+- When making the code, make *no or few assumptions* about the knowledge of the
+audience.
+- Keep the code *generalizable*. Our members come from diverse fields of
+research. What we share is the need to code.
+
+Lessons should be submitted as a Pull Request at least *one full day* before the
+session. Submitting the lessons as a Pull Request is also a great way to find
+out whether your lesson matches what is advertised in the Events repo.
+
+### Giving the lesson
+
+A few tips:
+
+- Start from the *very beginning*. Open the program or IDE (RStudio/Jupyter/etc)
```

Comment:
These first two apply to beginner lessons, but I think we do have some intermediate lessons where it's not quite so necessary.